### PR TITLE
document pre-created namespaces with labels

### PR DESCRIPTION
**Pull request by sparkoo**
Signed-off-by: Michal Vala <mvala@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
document how to prepare namespaces for users

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17842

### Specify the version of the product this PR applies to.
7.21

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 1]
```diff
@@ -94,3 +94,42 @@ endif::[]
...
+  name: admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: <username> <2>
+----
+<1> pre-created namespace
+<2> target user
+
+On Kubernetes, `che` ServiceAccount must have cluster-wide `list namespaces` privilege as well as `admin` role in target namespace.
```
Comment (metlos) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: Would this apply also on OpenShift without OAuth?

Comment (sparkoo) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: no, you can list projects with `che` SA on OpenShift. (you can't list namespaces, but we don't need that)

**Review by metlos — APPROVED** [review]

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc (hunk 1 above)
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]:
```suggestion
On {kubernetes}, `che` ServiceAccount must have a cluster-wide `list namespaces` privilege as well as an `admin` role in target namespace.
```

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 2]
```diff
@@ -94,3 +94,42 @@ endif::[]
 ----
 allowUserDefinedWorkspaceNamespaces
 ----
+
+== Pre-create {orch-namespace}s for users
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]:
```suggestion
== Pre-creating {orch-namespace}s for users
```

**Review by yhontyk — CHANGES_REQUESTED** [review]

**Comment by sparkoo** [comment]
@boczkowska commited your suggestions. Any other comments? From my side, it's ready to merge, the Che code is already in.

**Review by yhontyk — APPROVED** [review]
@sparkoo thanks, merging.

**Pull request merged**